_posts/2016-03-21-an-event-apart-nashville.md

```diff
@@ -0,0 +1,160 @@
+
+I wasn't quite sure what to expect going in, so I'd like to summarize my biggest
+takeaways and share some of the things I found exciting.
+
+### Major takeaways
+
+Knowing the kind of ground-breaking conversations that can happen at An Event
+Apart (for instance, the idea of responsive design was introduced by
+[Ethan Marcotte](http://ethanmarcotte.com/) at
+[An Event Apart Seattle 2010](http://aneventapart.com/news/post/five-years-ago-today-responsive-web-designs-debut))
+I was hoping for a glimpse into the future of the web, and I wasn't disappointed.
+Throughout the event, a few overarching themes came up.
+
+#### Unified design is the logical continuation of responsive design.
+
+Between the growing capabilities of devices, the sheer number of models, the
+differences in how we use them depending on age/location/race/preference/ability,
+and our ever-increasing use of small screens, there is no "mobile" and "desktop"
+anymore. In his talk, [Cameron Moll](http://cameronmoll.tumblr.com/) replaced
+the responsive design paradigm with **unified design** - a cohesive, consistent
+experience across all devices regardless of where the experience begins,
+continues, and ends. If we commit to this, we empower users to not only have a
+seamless experience using our products, we also allow them to use whichever
+device makes the most sense for them. As Cameron put it:
+
+> The best interface is the one within reach.
+
+Unified design isn't just important for the sake of a great user experience
+across multiple devices. Right now, 13% of US adults are smartphone-only users,
+and that percentage rises for people without a college degree, African Americans,
+people who make less than $20,000 a year, and young people -- 21% of millenials use the internet
+exclusively on mobile. And the US is behind most countries in mobile use! So if
+we continue gutting the functionality of our sites on small screens, we're
+penalizing large and sometimes vulnerable groups of users who can't afford or
+simply don't want another internet connection besides their phone.
+
+So why do our mobile sites fall so short? Constantly growing desktop screens
+gave us the opportunity to fill those screens with less-than-useful content. Once we
+needed to adapt to small screens, we had to cut down our desktop sites and often
+ended up sacrificing functionality. This is where the idea of mobile-first can
+save us - rather than designing for desktop then scaling down, we can design for
+mobile and then add enhancements depending on device capabilities. One of the
+biggest gains for me from this conference was finally grasping the true value
+of mobile-first design.
+
+#### Beyond devices, the Internet of Things is coming and can be awesome -- if we design responsibly.
+
+I'll be honest, the [Internet of Things](http://www.wired.com/2013/05/internet-of-things-2/)
+usually gives me a creepy,
+hair-standing-up-on-the-back-of-my-neck, Ray-Bradbury-was-right feeling. But the
```

Comment:
I didn't know who Ray Bradbury was. :embarrassed: Maybe link? Take it or leave it.